{"name": "test_delete_event_with_incorrect_id", "status": "broken", "statusDetails": {"message": "AttributeError: type object 'ConfigManager' has no attribute 'token'", "trace": "def test_delete_event_with_incorrect_id():\n>       response = test.api_tests.conftest.events_api.delete_wrong_event()\n\ntest_events_negative.py:29: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <api_service.events_api.EventsApi object at 0x1049b3230>, headers = None\nuse_default_token = True\n\n    def delete_wrong_event(self, headers=None, use_default_token=True):\n        if headers and use_default_token:\n            headers.update(ConfigManager.token)\n    \n        return (self.api_executor.get(\n            url=ConfigManager.delete_wrong_event_url,\n>           headers=headers or ConfigManager.token\n        ))\nE       AttributeError: type object 'ConfigManager' has no attribute 'token'\n\n../../api_service/events_api.py:66: AttributeError"}, "start": 1715117039988, "stop": 1715117039988, "uuid": "97913628-d6e2-489a-b79f-75ad0aa09129", "historyId": "39ecfea294607c69ac0ff9b5579b0d2e", "testCaseId": "39ecfea294607c69ac0ff9b5579b0d2e", "fullName": "test.api_tests.test_events_negative#test_delete_event_with_incorrect_id", "labels": [{"name": "parentSuite", "value": "test.api_tests"}, {"name": "suite", "value": "test_events_negative"}, {"name": "host", "value": "dariias-mbp.home"}, {"name": "thread", "value": "81500-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test.api_tests.test_events_negative"}]}